proto/minder/v1/minder.proto

```diff
@@ -90,8 +90,14 @@ extend google.protobuf.MethodOptions {
 message Cursor {
     // cursor is the index to start from within the collection being
     // retrieved. It's an opaque payload specified and interpreted on
-    // an per-rpc basis.
-    string cursor = 1;
+    // an per-rpc basis. Note it is base64 encoded.
```

Comment:
I'll update the comment because we do enforce the base64 part in the handler 👍

Reply:
My point was that saying "it's opaque" means it can be base64-encoded now, and some other pattern later.

Reply:
I'd leave the comment as-is, and enforce length + general character limits, rather than overspecifying on base64 now, and having clients break later if we change the design.